debian/endless-ca-cert.postinst

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+CONF_FILE=/etc/ca-certificates.conf
+CONF_FILE_NEW="$CONF_FILE".dpkg-new
+CONF_FILE_OLD="$CONF_FILE".dpkg-old
+CA_CONF=endless/endless-ca.crt
+
+if [ "$1" = configure ]; then
+    # Add the CA cert to the configuration file if there's no existing
+    # declaration
+    if [ -f "$CONF_FILE" ]; then
+        cp "$CONF_FILE" "$CONF_FILE_NEW"
+        if ! grep -q "^!\?${CA_CONF}$" "$CONF_FILE_NEW"; then
+            echo "$CA_CONF" >> "$CONF_FILE_NEW"
+        fi
+        if cmp -s "$CONF_FILE" "$CONF_FILE_NEW"; then
+            rm -f "$CONF_FILE_NEW"
+        else
+            mv -f "$CONF_FILE" "$CONF_FILE_OLD"
+            mv "$CONF_FILE_NEW" "$CONF_FILE"
+        fi
+    fi
+fi
+
+#DEBHELPER#
```

Comment:
Might be good to have a longer comment why you need the exit 0

Reply:
It's just standard practice for maintainer scripts - https://www.debian.org/doc/debian-policy/#exit-status. Otherwise we'd just use the status of the last command that debhelper substituted. The `#DEBHELPER#` is actually a template that it substitutes with other automatic maintainer script actions.

Reply:
k